Homework8/Task2/ExceptionGenerator.cs

```diff
@@ -0,0 +1,17 @@
+﻿namespace Homework8.Task2
+{
+    public class ExceptionGenerator
+    {
+        public void GenerateException()
```

Comment:
In the task said - method should throw an exception but not to handle

Reply:
Fixed